helix_fhir_client_sdk/dictionary_parser.py

```diff
@@ -1,20 +1,20 @@
-from typing import List, Any, Dict, Union
+from typing import Any
 
 
 class DictionaryParser:
     def __init__(self) -> None:
         pass
 
     @staticmethod
-    def flatten(my_list: List[Any]) -> List[Any]:
+    def flatten(my_list: list[Any]) -> list[Any]:
```

Comment:
I like this switching to the native types we should do this across our code base and remove dependency on typing.

Reply:
ruff does it automatically for the codebase :-)